### PR TITLE
pkp/pkp-lib#1683 Implement OJS2.4.x -> OJS 3.0 migration

### PR DESCRIPTION
**Pull request by asmecher**

### CODE REVIEW

File: classes/submission/reviewer/form/ReviewerReviewStep3Form.inc.php [hunk 1]
```diff
@@ -42,11 +42,11 @@ function initData() {
 		// Retrieve most recent reviewer comments, one private, one public.
 		$submissionCommentDao = DAORegistry::getDAO('SubmissionCommentDAO');
 
-		$submissionComments = $submissionCommentDao->getReviewerCommentsByReviewerId($reviewAssignment->getReviewerId(), $reviewAssignment->getSubmissionId(), $reviewAssignment->getId(), true);
+		$submissionComments = $submissionCommentDao->getReviewerCommentsByReviewerId($reviewAssignment->getSubmissionId(), $reviewAssignment->getReviewerId(), $reviewAssignment->getId(), true);
 		$submissionComment = $submissionComments->next();
 		$this->setData('comment', $submissionComment?$submissionComment->getComments():'');
```
Comment (bozana) [inline classes/submission/reviewer/form/ReviewerReviewStep3Form.inc.php]: shouldn't this be called 'comments' now?

File: classes/submission/reviewer/form/ReviewerReviewStep3Form.inc.php [hunk 2]
```diff
@@ -42,11 +42,11 @@ function initData() {
...
 
-		$submissionCommentsPrivate = $submissionCommentDao->getReviewerCommentsByReviewerId($reviewAssignment->getReviewerId(), $reviewAssignment->getSubmissionId(), $reviewAssignment->getId(), false);
+		$submissionCommentsPrivate = $submissionCommentDao->getReviewerCommentsByReviewerId($reviewAssignment->getSubmissionId(), $reviewAssignment->getReviewerId(), $reviewAssignment->getId(), false);
 		$submissionCommentPrivate = $submissionCommentsPrivate->next();
 		$this->setData('commentPrivate', $submissionCommentPrivate?$submissionCommentPrivate->getComments():'');
```
Comment (bozana) [inline classes/submission/reviewer/form/ReviewerReviewStep3Form.inc.php]: and this 'commentsPrivate'?

File: classes/submission/reviewer/form/ReviewerReviewStep3Form.inc.php (hunk 1 above)
Comment (asmecher) [inline classes/submission/reviewer/form/ReviewerReviewStep3Form.inc.php]: Nope, the Reviewer's interface still just shows the most recent comment. This is kind of a legacy data concern -- in OJS2 multiple comments were allowed, and in OJS3 only one is.

Comment (bozana) [inline classes/submission/reviewer/form/ReviewerReviewStep3Form.inc.php]: ah, ok, yes, yes... sorry...

**Pull request merged**